web/docs/interfaces/Fields.md

```diff
@@ -218,6 +354,14 @@ Timestamp when this flow was received and processed by the flow collector, in se
 
 ___
 
+### TimeFlowRttNs
```

Comment:
Will RTT be included in the product docs regeneration? 🤔

Reply:
Yes, every field is present here even if it's feature related. However we may add a mention like `*Dev Preview*` in between to clarify it.